### PR TITLE
Create Proposal Modal

### PR DESCRIPTION
**Pull request by stefanpetkov90**

### CODE REVIEW

File: src/components/Dropdown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { FormControl, Select, MenuItem, SelectChangeEvent } from '@mui/material'
+import ArrowIcon from '../../assets/vectors/arrow-down-blue.svg'
```
Comment (stanfilip0v) [inline src/components/Dropdown/index.tsx]: can be imported without '../'

File: src/containers/Proposals/components/ProposalModal/Failure.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { Box, Typography, Button } from '@mui/material'
+import { initialModalState, ModalProps } from 'store/proposals'
+
+import FailureIcon from '../../../../assets/vectors/failure.svg'
```
Comment (stanfilip0v) [inline src/containers/Proposals/components/ProposalModal/Failure.tsx]: can be imported without '../'

File: src/containers/Proposals/components/ProposalModal/Success.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
+import {
+  InfoRounded as InfoRoundedIcon,
+  OpenInNewRounded as OpenInNewRoundedIcon
+} from '@mui/icons-material'
+import { Box, Typography, Divider, Stack, Button } from '@mui/material'
+import { initialModalState, ModalProps } from 'store/proposals'
+
+import { useSelector } from 'react-redux'
+import { RootState } from 'store'
+import SuccessIcon from '../../../../assets/vectors/success.svg'
```
Comment (stanfilip0v) [inline src/containers/Proposals/components/ProposalModal/Success.tsx]: can be imported without '../'

**Review by SpaghettiOverload — APPROVED** [review]

File: src/containers/Proposals/index.tsx [hunk 4]
```diff
@@ -1,11 +1,17 @@
 import { Box, InputAdornment, Typography, Chip, Button } from '@mui/material'
 import Card from 'components/Card/Card'
+import { ProposalStatus } from 'store/proposals'
 import SearchIcon from 'assets/vectors/search-icon.svg'
+import CrossIcon from '../../assets/vectors/cross-blue.svg'
```
Comment (stanfilip0v) [inline src/containers/Proposals/index.tsx]: can be imported without '../'

**Review by stanfilip0v — APPROVED** [review]

File: src/components/Dropdown/index.tsx (hunk 1 above)
Comment (stefanpetkov90) [reply src/components/Dropdown/index.tsx]: Done

File: src/containers/Proposals/components/ProposalModal/Failure.tsx (hunk 2 above)
Comment (stefanpetkov90) [reply src/containers/Proposals/components/ProposalModal/Failure.tsx]: Done

File: src/containers/Proposals/components/ProposalModal/Success.tsx (hunk 3 above)
Comment (stefanpetkov90) [reply src/containers/Proposals/components/ProposalModal/Success.tsx]: Done

File: src/containers/Proposals/index.tsx (hunk 4 above)
Comment (stefanpetkov90) [reply src/containers/Proposals/index.tsx]: Done

**Pull request merged**